tests/unit/dependabot_alerts/cli_test.py

```diff
@@ -1,23 +1,10 @@
-from importlib.metadata import version
-
 import pytest
 
 from dependabot_alerts.cli import cli
 
 
-def test_it():
-    cli([])
```

Comment:
This test works fine with the cookiecutter's auto-generated do-nothing CLI but with a CLI that actually does stuff I would have to put some mocks in place in order to unit test the CLI, otherwise it ends up calling a bunch of code (and APIs!) and crashes. Just deleted the test